zkevm-circuits/src/evm_circuit/param.rs

```diff
@@ -36,3 +36,7 @@ pub(crate) const N_BYTES_GAS: usize = N_BYTES_U64;
 
 // Number of bytes that will be used for call data's size.
 pub(crate) const N_BYTES_CALLDATASIZE: usize = N_BYTES_U64;
+
+// The max number of bytes that can be copied in a step limited by the number
+// of cells in a step
+pub(crate) const MAX_COPY_BYTES: usize = 71;
```

Comment:
A minor suggestion: MAX_COPY_BYTES is not something independent, but calculated  as (STEP_HEIGHT * STEP_WIDTH - N_CELLS_STEP_STATE - ?) / (cost of one copy step?) . So if we adjust STEP_HEIGHT later, MAX_COPY_BYTES should be changed accordingly?   It is not an issue directly for this PR, but we may fix it later.

Reply:
With `CALLDATACOPY` and `CODECOPY` we've used `MAX_COPY_BYTES = 32`, and import it from `bus_mapping::constants::MAX_COPY_BYTES`.

I think it will be good to re-use that.

Reply:
@roynalnaruto   oh, why change MAX_COPY_BYTES to 32 now? here use old value from memory copy :),   but good to me to re-use latest bus_mapping::constants::MAX_COPY_BYTES!

Reply:
> 

@lispc thanks you mention this, let keep an eye on for after variable height step pr merge!

Reply:
updated in [c8342ff](https://github.com/appliedzkp/zkevm-circuits/pull/335/commits/c8342ff0df7ef9cfb22d2a55c05ce45ee318ccd6) thanks